docs/en/latest/FAQ.md

```diff
@@ -76,3 +76,13 @@ kubectl get crd apisixroutes.apisix.apache.org -o jsonpath='{ .spec.versions[*].
 If you can not find `v2beta3` in `ApisixRoute` definition file. Please apply the latest version of `ApisixRoute`.
 
 Ref to FAQ #7.
+
+### 9. Modify the Admin API key in APISIX-Ingress
+
+Usually, you need to modify the Admin API key to protect Apache APISIX and APISIX Ingress. Please refer to this [link](https://apisix.apache.org/zh/docs/apisix/how-to-build/) to simply change Apache APISIX.
+
+However, in apisix-ingress-controller, if we need to change the Admin API key, we also need to change the Admin API key in apisix-ingress-controller. There are two ways to implement the requirements here.
+
+In one method, we need to change the values of the two values.yaml files in the helm chart We need to change the Admin API key values in both helm chart packages.
```

Comment:
This sentence makes me confused. Which values should we modify?

Reply:
Let me refine it a little～

Reply:
> This sentence makes me confused. Which values should we modify?

+1. 

If the description is not clear, please paste the specific code directly, it will be clearer

Reply:
I have changed the description of the first method.
Please review it, sir.